content/agent/kubernetes/host_setup.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Kubernetes Host Setup
+kind: documentation
+further_reading:
+- link: "agent/autodiscovery"
+  tag: "Documentation"
+  text: Docker Agent Autodiscovery
+- link: "agent/kubernetes/host_setup"
+  tag: "documentation"
+  text: "Kubernetes Host Setup"
+- link: "agent/kubernetes/integrations"
+  tag: "documentation"
+  text: "Custom Integrations"
+---
+
+Installing the Agent directly on your host (rather than having the Agent run in a Pod) provides additional visibility into your ecosystem, independent of Kubernetes.
+
+To gather your kube-state metrics:
+
+1. Download the [Kube-State manifests folder][3].
+
+2. Apply them to your Kubernetes cluster:
+  ```
+  kubectl apply -f <NAME_OF_THE_KUBE_STATE_MANIFESTS_FOLDER>
+  ```
+
+## Setup Kubernetes DNS
+### Configuration
+
+Since [Agent v6][4], Kubernetes DNS integration works automatically with the [Autodiscovery][5].
+
+- Please note that these metrics are unavailable for Azure Kubernetes Service (AKS) at this point in time.  
+
+## Collect container logs
+
+**Available for Agent >6.0**
+
+Two installations are possible:
+
+- On the node where the agent is external to the Docker environment
+- Deployed with its containerized version in the Docker environment
+
+Take advantage of DaemonSets to [automatically deploy the Datadog Agent on all your nodes][14]. Otherwise follow the [container log collection steps][15] to start collecting logs from all your containers.
+
+## Further Reading
+To get a better idea of how (or why) to integrate your Kubernetes service, check out our [series of blog posts][13] about it.
+
+[1]: https://hub.docker.com/r/datadog/agent/
+[2]: /agent/basic_agent_usage/kubernetes
+[3]: https://github.com/kubernetes/kube-state-metrics/tree/master/kubernetes
```

Comment:
I'm tempted to leave this one point to master because pinning it to a tagged version means we'll point to an outdated version as soon as they upgrade, and on the other hand there's very little chance that the `kubernetes` folder go away. I think it's fine to leave this one as it is.